figma-shop/src/app/app-routing.module.ts

```diff
@@ -1,8 +1,22 @@
 import { NgModule } from '@angular/core';
```

Comment:
Could you please make sure that all TS Lint issues are fixed in your project?
Could you please rename PR on some normal title (it should be taken from a branch name)?
`Create skeleton for application`

Reply:
Configured TSLint